multiclusterhub/charts/search-prod/templates/aggregator-configmap.yaml

```diff
@@ -0,0 +1,17 @@
+# Copyright (c) 2020 Red Hat, Inc.
+# Copyright Contributors to the Open Cluster Management project
+
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: search-aggregator
+  labels:
+    config: acm-proxyserver
+data:
+  service: "{{ .Release.Namespace }}/search-aggregator"
+  port: "3010"
+  path: "/aggregator/clusters/"
+  sub-resource: "/sync"
+  use-id: "true"
+  secret: "{{ .Release.Namespace }}/search-aggregator-certs"
+  caConfigMap: search-ca-crt
```

Comment:
Missing newline at end of last line.

https://stackoverflow.com/a/729795/1437822